src/coreComponents/physicsSolvers/python/PySolver.cpp

```diff
@@ -95,10 +95,29 @@ static PyObject * reinit( PySolver * self, PyObject *args )
   Py_RETURN_NONE;
 }
 
+static PyObject * cleanup( PySolver * self, PyObject *args )
+{
+  VERIFY_NON_NULL_SELF( self );
+  VERIFY_INITIALIZED( self );
+  GEOSX_UNUSED_VAR( args );
+
+  double time;
+  if( !PyArg_ParseTuple( args, "d", &time ) )
+  {
+    return nullptr;
+  }
+
+  geosx::DomainPartition & domain = self->group->getGroupByPath< DomainPartition >( "/Problem/domain" );
+  self->group->cleanup( time, 0, 0, 0.0, domain );
```

Comment:
Shouldn't we give real value for `dt` and `cycleNumber`  to `cleanup()` instead of 0 ?